testing/data/DATA_LICENSES.md

```diff
@@ -10,6 +10,7 @@
 - Cameras.gltf: glTF-Sample-Models: Public Domain
 - cow.vtk: VTK Data: BSD
 - cow.vtp: VTK Data: BSD
+- DFLIYX1B.TTF: [fontsaddict Free license font](http://www.fontsaddict.com/font/dfliyuanxbold1b.html)
```

Comment:
Unless I'm mistaken, license in not super clear on the website and list this font as "Free for personnal use". Not ok to add in this context I'm afraid.